dev/source/docs/building-setup-windows-eclipse.rst

```diff
@@ -41,7 +41,7 @@ Once Eclipse has been started import ArduPilot by doing the following:
     :width: 450px
 
 - In the Project Explorer tab on the top right, right-mouse-button-click on the "ardupilot" folder and select "Properties"
-- When the "Properties for ardupilot" window appears, under "C/C++ Build", uncheck "Use default build command" and enter ``c:\cygwin64\bin\python2.7 waf`` into the "Build command" field as shown below
+- When the "Properties for ardupilot" window appears, under "C/C++ Build", uncheck "Use default build command" and enter ``c:\cygwin64\bin\python3.6 waf`` into the "Build command" field as shown below
```

Comment:
It has been a while since I used this, but I just double checked. For my Eclipse install I must use the following, `c:\cygwin64\bin\python3.6m waf` in order for it to find python.

Reply:
@hendjoshsr71 Thank you for the update. I changed PR to 3.6m instead of 3.6.